src/control/control.h

```diff
@@ -98,27 +106,28 @@ class ControlDoublePrivate : public QObject {
     void setValueFromMidi(MidiOpCode opcode, double dParam);
     double getMidiParameter() const;
 
-    inline bool ignoreNops() const {
+    bool ignoreNops() const {
         return m_bIgnoreNops;
     }
 
-    inline void setDefaultValue(double dValue) {
+    void setDefaultValue(double dValue) {
         m_defaultValue.setValue(dValue);
     }
 
-    inline double defaultValue() const {
+    double defaultValue() const {
         return m_defaultValue.getValue();
     }
 
-    inline ControlObject* getCreatorCO() const {
-        return m_pCreatorCO;
+    ControlObject* getCreatorCO() const {
+        return m_pCreatorCO.loadAcquire();
```

Comment:
We have here a potential race condition. 

Here we give away raw pointer that is no longer controlled and may outlive the object it is referencing. 
I think that does not really happen, but it is used in: ControlObject::getControl()
We should work to eliminate that. 
It should be replaced by ControlObject::get() and ControlObject::set(). 

Out of scope of this PR.

Reply:
I am not able to fix all the bad design decisions from the past. Out of scope.

Reply:
I just want to point out that I was about to start, but our discussion was stalled. 
We need to continue at one point.